docs/getting-started/getting-started-with-cwl.rst

```diff
@@ -98,6 +98,8 @@ repository:
         dct: http://purl.org/dc/terms/
         foaf: http://xmlns.com/foaf/0.1/
 
+.. note:: Dockstore supports registering tools and workflows written in the `sbg:draft-2 <https://docs.sevenbridges.com/docs/about-the-common-workflow-language#sbgdraft-2>`__ implementation of CWL (yaml version only) for the Seven Bridges Platform.
```

Comment:
1) What would the author list as the CWL version in the SB implementation of CWL? `sbg:draft-2`? I think that needs clarifying.

2) It seems like SB is suggesting that users test and update their workflows to be more interoperable rather than directly hosting (see their new documentation: https://sb-biodatacatalyst.readme.io/docs/maintaining-and-versioning-cwl-on-external-tool-repositories). I think we can give the user both options or defer to what SB is suggesting -- any opinions? 

Maybe instead say:
"Dockstore supports registering tools and workflows written in the `sbg:draft-2 <https://docs.sevenbridges.com/docs/about-the-common-workflow-language#sbgdraft-2>`__ implementation of CWL (yaml version only) that is optimized for the Seven Bridges cloud-based platform. The workflow version in the descriptor file would be listed as `sbg:draft-2`. 

However, this implementation is less portable. Seven Bridges provides [instructions](https://sb-biodatacatalyst.readme.io/docs/maintaining-and-versioning-cwl-on-external-tool-repositories) for how to transition tools and workflows developed in the Seven Bridges Software Development Kit to GitHub for publishing in Dockstore."

Reply:
Reading more of https://docs.sevenbridges.com/docs/about-the-common-workflow-language, yes I think the cwl version would be `sbg:draft-2` but `CWL v1.0` natively supports the custom extensions in `sbg:draft-2`. I imagine 1.1 and 1.2 support the extensions as well, but it doesn't explicitly say so in the doc.

Reply:
Looks good, but did you want to include the link to the SB instructions as well?

Reply:
As a wise man once said `d'oh!`. I was also wrong about our version support. d'oh x2

Reply:
:)